test/scripts/install-contour-working.sh

```diff
@@ -44,15 +44,13 @@ if ! kind::cluster::exists "$CLUSTERNAME" ; then
     exit 2
 fi
 
-# Set (pseudo) random image tag to trigger restarts at every deployment.
-# TODO: Come up with a scheme that doesn't fill up the dev environment with randomly-tagged images.
-VERSION="v$$"
+docker rmi ghcr.io/projectcontour/contour:latest --force
```

Comment:
Since `latest` is a [real tag that we publish](https://github.com/projectcontour/contour/pkgs/container/contour/138343235?tag=latest), I'd suggest using something like `dev` or `working` instead.